go/client/getentries.go

```diff
@@ -28,8 +32,21 @@ func rawGetEntries(ctx context.Context, httpClient *http.Client, logURL string,
 	if end < start {
 		return nil, errors.New("start should be <= end")
 	}
+
+	baseURL, err := url.Parse(strings.TrimRight(logURL, "/") + GetEntriesPath)
+	if err != nil {
+		return nil, err
+	}
+
+	baseURL.RawQuery = url.Values{
+		"start": []string{strconv.FormatInt(start, 10)},
```

Comment:
It seems like it's one of those things like `strtoul` that you have to look up every damn time you see it... There was a straightforward-looking `Itoa`, but it takes an `int`, not an `int64`. :stuck_out_tongue_closed_eyes: